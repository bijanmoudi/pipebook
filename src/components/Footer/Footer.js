import React from 'react';
import classnames from 'classnames';
import './index.scss';

const Footer = ({ className, ...restProps }) => {
	return (
		<footer
			className={classnames('footer', { [className]: className })}
			{...restProps}
		></footer>
	);
};

export default Footer;
