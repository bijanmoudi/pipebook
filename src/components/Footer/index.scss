/*=====================================
=            Footer Styles            =
=====================================*/

@import '../../assets/styles/helpers/colors';
@import '../../assets/styles/helpers/units';

.footer {
	padding: rem(28) 0;
	background-color: get-color('gray-light-very');
	box-shadow: 0 rem(-1) rem(3) get-color('gray-light', 0.45);
}
