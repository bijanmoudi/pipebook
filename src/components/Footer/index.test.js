import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

describe('Testing "Footer" component', () => {
	it('Rendering the footer', () => {
		const tree = renderer.create(<Footer />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
