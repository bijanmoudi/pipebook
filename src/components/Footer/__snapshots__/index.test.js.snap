// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing "Footer" component Rendering the footer 1`] = `
<footer
  className="footer"
/>
`;
